src/client.ts

```diff
@@ -1,23 +1,50 @@
 import { GraphQLClient } from 'graphql-request';
 import { gql } from 'graphql-tag';
-import { VALID_SUBSQUID_URLS } from './networks';
+import {
+  NetworkName,
+  ETHEREUM_URL,
+  ETHEREUM_SEPOLIA_URL,
+  BSC_URL,
+  POLYGON_URL,
+  ARBITRUM_URL,
+} from './networks.js';
 
 // Import generated types
 import type { Query } from './generated/types';
 
-export const isNetworkValid = (url: string): boolean => {
-  return VALID_SUBSQUID_URLS.includes(url);
+// export const isNetworkValid = (url: string): boolean => {
```

Comment:
Is there any chance we might be leaving this commented ? We can remove it right ?